python/paddle/fluid/tests/unittests/mkldnn/test_activation_mkldnn_op.py

```diff
@@ -19,7 +19,7 @@
 from scipy.special import expit
 import paddle.fluid.core as core
 from paddle.fluid.tests.unittests.op_test import OpTest, convert_float_to_uint16
-from paddle.fluid.tests.unittests.test_activation_op import TestActivation, TestRelu, TestTanh, TestSqrt, TestAbs, TestLeakyRelu, TestSwish, TestRelu6, TestSigmoid
+from paddle.fluid.tests.unittests.test_activation_op import TestActivation, TestRelu, TestTanh, TestSqrt, TestAbs, TestLeakyRelu, TestSwish, TestHardSwish, TestRelu6, TestSigmoid
```

Comment:
Consider following formatting to prevent extremely long lines.
```suggestion
from paddle.fluid.tests.unittests.test_activation_op import (
    TestActivation,
    TestRelu,
    TestTanh,
    TestSqrt,
    TestAbs,
    TestLeakyRelu,
    TestSwish,
    TestHardSwish,
    TestRelu6,
    TestSigmoid,
)
```